src/ActivityLogger.php

```diff
@@ -30,18 +30,21 @@ class ActivityLogger
     /** @var \Illuminate\Support\Collection */
     protected $properties;
 
+    /** @var string */
+    protected $authDriver;
```

Comment:
Because we now assign the `$authDriver` on the instance so we can use it in multiple methods.  Keep or remove?